sagemaker-python-sdk/dgl_lifesci_bap/pytorch_bap.ipynb

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
<p>Grammar nit - use <code>Optional</code> rather than <code>Option</code> ?</p><p> </p><p>Typo - <code>pecify</code>  -&gt; <code>specify</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Changed the statement